demos/demo_tricamera.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
The file is not PEP8-compliant.  If you want to give `black` a try, limit the line length to 79, i.e.

    black --line-length 79 filename.py

Reply:
Thank you :) ! Sorry hadn't done this earlier.